src/dotnet/APIView/APIViewWeb/Controllers/PullRequestController.cs

```diff
@@ -29,10 +32,33 @@ public PullRequestController(PullRequestManager pullRequestManager, ILogger<Auto
             string commitSha,
             string repoName)
         {
+            if (!ValidateInputParams())
+            {
+                return StatusCode(StatusCodes.Status400BadRequest);
+            }
+
             await _pullRequestManager.DetectApiChanges(buildId, artifactName, filePath, pullRequestNumber, commitSha, repoName);
```

Comment:
Should we go ahead and just do the splitting of the repoName into 2 fields here?

Reply:
repo owner and repo name combination is also stored in cosmos DB and we need this combo to search a specific PR in a repo. Making it two different fields is not going to help in cosmos DB fetch but rather will make query expensive due to one more column in the future if volume of data is high.